src/searchbar.cpp

```diff
@@ -24,9 +24,11 @@ void SearchButton::set_searchMode(bool searchMode)
     } else {
         auto kiwixApp = KiwixApp::instance();
         if (kiwixApp->isCurrentArticleBookmarked()) {
-            setIcon(QIcon(":/icons/reading-list-active.svg"));
+            setIcon(QIcon(":/icons/star.svg"));
+            setToolTip(gt("Add to the reading list"));
```

Comment:
Move this message into `resources/i18n/en.json` and update `resources/i18n/qqq.json`.